yt_dlp/extractor/hungama.py

```diff
@@ -71,14 +74,14 @@ class HungamaSongIE(InfoExtractor):
     _VALID_URL = r'https?://(?:www\.)?hungama\.com/song/[^/]+/(?P<id>\d+)'
     _TEST = {
         'url': 'https://www.hungama.com/song/kitni-haseen-zindagi/2931166/',
-        'md5': 'a845a6d1ebd08d80c1035126d49bd6a0',
+        'md5': 'd4a6a05a394ad0453a9bea3ca00e6024',
         'info_dict': {
             'id': '2931166',
-            'ext': 'mp4',
+            'ext': 'mp3',
             'title': 'Lucky Ali - Kitni Haseen Zindagi',
             'track': 'Kitni Haseen Zindagi',
             'artist': 'Lucky Ali',
-            'album': 'Aks',
+            'album': '',
```

Comment:
why is this an empty string? If not found, it should be None

Reply:
this is still the old test url and was proly modified at some point by the artist to change this to empty string
So the albums is literally and empty string in the json
Using str_or_none won't fix tihs.
```
>>> str_or_none('')
''
```
If u still want it to be `None` then i can surely fix it.

Reply:
If it were fields like `description`, it would make sense to have `''` separate from `None`. but it doesn't really makes sense here. So maybe add an `or None`.